README.md

```diff
@@ -4,6 +4,54 @@
 
 Kepler (Kubernetes-based Efficient Power Level Exporter) is a Prometheus exporter that measures energy consumption metrics at the container, pod, and node level in Kubernetes clusters.
 
+## 🚀 Major Rewrite: Kepler (0.10.0 and above)
+
+**Important Notice:** Starting with version 0.10.0, Kepler has undergone a complete ground-up rewrite.
+This represents a significant architectural improvement while maintaining the core mission of
+accurate energy consumption monitoring for cloud-native workloads.
+
+> 📢 **Read the full announcement:** [CNCF Slack Announcement](https://cloud-native.slack.com/archives/C05QK3KN3HT/p1752049660866519)
```

Comment:
I think we should replace this with the release link in the future or even change to blog post in CNCF about this rewrite.

Reply:
I love the idea of a blog post on CNCF !